src/data/move.ts

```diff
@@ -261,6 +257,18 @@ export abstract class Move implements Localizable {
     return false;
   }
 
+  isAttackMove(): this is AttackMove {
+    return this.category === MoveCategory.PHYSICAL || this.category === MoveCategory.SPECIAL;
+  }
+
+  isStatusMove(): this is StatusMove {
+    return false;
```

Comment:
```suggestion
    return this.category === MoveCategory.STATUS;
```
(If desired, also add a check that the move's target is not MoveTarget.USER. See #394)